modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -468,6 +468,26 @@ TEST_F(ONNXtest, InferMultOutput)
     // Validate
     validate();
 }
+
+TEST_F(ONNXClassificationTest, InferPaddingCHW)
+{
+    // FIXME: Remove this test and add padding to test with FRCNN
+    useModel("classification/squeezenet/model/squeezenet1.0-9");
+    in_mat1 = initMatrixRandU(CV_8UC3, cv::Size{212, 200});
+
+    G_API_NET(SqueezNet, <cv::GMat(cv::GMat)>, "squeeznet");
```

Comment:
`squeeze` :)

Reply:
Abbreviation) I'll take a look at this in last PR.